guiclient/enterPoReceipt.cpp

```
@@ -297,9 +298,10 @@ void enterPoReceipt::sPost()
       int result = postLine.value("result").toInt();
       if (result < 0 && result != -11) // ignore -11 as it just means there was no inventory
       {
+        ErrorReporter::error(QtCriticalMsg, this, tr("Error Posting P/O Receipt Information"),
+                               storedProcErrorLookup("postReceipt", result),
+                               __FILE__, __LINE__);
```

Please report the error after the rollback here and at ~330.